packages/contract/test/AccountVerify1271.t.sol

```diff
@@ -67,11 +89,64 @@ contract AccountVerify1271Test is Test {
         // Fix the signature by changing s
         uint256 n = 0xFFFFFFFF00000000FFFFFFFFFFFFFFFFBCE6FAADA7179E84F3B9CAC2FC632551;
         s = n - s;
-        sig = abi.encodePacked(hex"00", r, s); // keySlot, r, s
+        sig = abi.encodePacked(
+            uint8(0), // keySlot
+            abi.encode( // signature
+                Utils.rawSignatureToSignature({
+                    challenge: abi.encodePacked(
+                        bytes32(
+                            0x15fa6f8c855db1dccbb8a42eef3a7b83f11d29758e84aed37312527165d5eec5
+                        )
+                    ),
+                    r: 0x3f033e5c93d0310f33632295f64d526f7569c4cb30895f50d60de5fe9e0e6a9a,
+                    s: s
+                })
+            )
+        );
         console.log("fixed sig s:", s);
 
         // Now it's accepted
         ret = account.isValidSignature(hash, sig);
         assertEq(ret, bytes4(0x1626ba7e)); // ERC1271_MAGICVALUE
     }
+
+    // TODO
```

Comment:
still worth doing?

Reply:
deleted for this PR, will look in other PR